lib/aggregation/accumulator/src/lib/reducer-config.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+const { map } = require('underscore');
+const seqid = require('abacus-seqid');
+const urienv = require('abacus-urienv');
+// Resolve service URIs
+const uris = urienv({
+  auth_server: 9882,
+  aggregator: 9300
+});
+
+// Return OAuth system scopes needed to write input docs
+const iwscope = (secured) => (udoc) =>
+  secured()
+    ? { system: ['abacus.usage.write'] }
+    : undefined;
+
+// Return OAuth system scopes needed to read input and output docs
+const rscope = (secured) => (udoc) =>
+  secured()
+    ? { system: ['abacus.usage.read'] }
+    : undefined;
+
+// Return the keys and times of our docs
+const ikey = (udoc) => [
+  udoc.organization_id,
+  udoc.resource_instance_id,
+  udoc.consumer_id || 'UNKNOWN',
+  udoc.resource_id,
+  udoc.plan_id
+].join('/');
+
+const itime = () => seqid();
+
+const igroups = (udoc) => [
+  [
+    udoc.organization_id,
+    udoc.resource_instance_id,
+    udoc.consumer_id || 'UNKNOWN',
+    udoc.plan_id,
+    udoc.metering_plan_id,
+    udoc.rating_plan_id,
+    udoc.pricing_plan_id
+  ].join('/')
+];
+
+const okeys = (udoc) => {
+  const firstKey = [
```

Comment:
Maybe use better names than `firstKey` and `secondKey`, if you know the purpose behind those keys. For example, `secondKey` could be renamed to `markerDocKey` or something of the kind.